FEM/pcs_dm.cpp

```diff
@@ -772,21 +772,18 @@ double CRFProcessDeformation::Execute(int loop_process_number)
 
 				Error = Norm / InitialNorm;
 				ErrorU = NormU / InitialNormU0;
-				// Error = Norm / InitialNorm;
-				// ErrorU = NormU / InitialNormU0;
-				// if(Norm < Tolerance_global_Newton && Error > Norm)
-				//	Error = Norm;
-				//           if(Norm<TolNorm)  Error = 0.01*Tolerance_global_Newton;
-				// if((NormU / InitialNormU) <= Tolerance_global_Newton)
-				//	Error = NormU / InitialNormU;
+				if(Norm < Tolerance_global_Newton && Error > Norm)
+				    Error = Norm;
+
+				if((NormU / InitialNormU) <= Tolerance_global_Newton)
+					Error = NormU / InitialNormU;
 
 				// Compute damping for Newton-Raphson step
 				damping = 1.0;
-				//           if(Error/Error1>1.0e-1) damping=0.5;
-// if(Error / Error1 > 1.0e-1 || ErrorU / ErrorU1 > 1.0e-1)
-// damping = 0.5;
-// if(ErrorU < Error)
-//	Error = ErrorU;
+				if (Error / Error1 > 1.0e-1 || ErrorU / ErrorU1 > 1.0e-1)
+					damping = 0.5;
```

Comment:
Whether this is allowed to occur should be input file controlled. Sometimes these solutions work, sometimes they don't. At any case, it makes it difficult to verify a convergence rate if I don't have control over these things. Btw, TH2M runs fine now if I only test the relative residual norm. Even without damping.